### DISCUSSION TITLE
Male TTS

### DISCUSSION THREAD

**Discussion by nellorebhanuteja** [espnet/espnet#4367] [Q&A] [answered]
@kan-bayashi 
Is there a TTS model available with male voice ?
FastSpeech2 +. HiFIGAN

**Comment by kan-bayashi** [comment, marked as answer]
What language do you want?

**Reply by nellorebhanuteja** [reply]
English

**Reply by roholazandie** [reply]
I have developed one male speech model. Here you can have access to the models:

https://huggingface.co/espnet/english_male_ryanspeech_conformer_fastspeech2
https://huggingface.co/espnet/english_male_ryanspeech_fastspeech2
https://huggingface.co/espnet/english_male_ryanspeech_tacotron
https://huggingface.co/espnet/english_male_ryanspeech_fastspeech

And the dataset is available here:
https://huggingface.co/datasets/Roh/ryanspeech

with the demo:
https://huggingface.co/spaces/Roh/RyanSpeech

If you want high-quality results you should use our trained vocoder here:
https://drive.google.com/file/d/10GYvB_mIKzXzSjD67tSnBhknZRoBjsNb/view

**Reply by nellorebhanuteja** [reply]
thank you!

**Reply by alexxony** [reply]
korean male

**Comment by nellorebhanuteja** [comment]
hi @roholazandie

in you Interspeech paper, you mentioned the dataset size is 10 hrs.
the quality is impressive
can you say, which recipes did you use ?

**Reply by roholazandie** [reply]
Hi,
I used the ljspeech recipes. I tried to convert all my data formats to ljspeech, if you look in the dataset you see it is exactly formatted as ljspeech.

**Reply by nellorebhanuteja** [reply]
right ...
